### ISSUE TITLE
Error when defining a class in a macro

### ISSUE THREAD

**Issue by ConjuringCoffee** [SAP/abap-cleaner#93]
Hi Jörg-Michael, I've been messing around with macros and found a case that can't be handled by the ABAP Cleaner.

```abap
DEFINE define_exception.
  CLASS &1 DEFINITION FOR TESTING INHERITING FROM cx_static_check.
  ENDCLASS.


  CLASS &1 IMPLEMENTATION.
  ENDCLASS.
END-OF-DEFINITION.
```

The following error is logged:
`Parse error in line 5: expected END-OF-DEFINITION, but found CLASS. Opening command (line 3): DEFINE define_exception.`

EDIT: I love how even the GitHub syntax highlighting is confused 😄

**Comment by ConjuringCoffee** [comment]
PS: I'm not going to include this in any code that will go to production, so this has low priority for me.

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

nice finding! Although I regularly run ABAP cleaner on dozens of megabytes of code for testing, I guess this never came up so far, because ```CLASS``` was always found on top level. Will be fixed in the next release!

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

thanks again for this finding! This should now be fixed with version 1.5.3, which was just released.

Kind regards,
Jörg-Michael
